webscraper.js

```diff
@@ -9,22 +9,26 @@ build();
 if (require.main === module) {
   //Get folders
   const dirs = p => readdirSync(p).filter(f => statSync(p+"/"+f).isDirectory());
-  const folders = dirs(join(dirname(__dirname), 'product_classification_test_data'));
+  const folders = dirs(join(dirname(__dirname), 'fathom-products' , 'product_classification_test_data'));
+  driver.manage().window().maximize();
```

Comment:
Will this make the window size depend on the resolution of the main monitor or something? I could see page elements thus reporting different sizes on different machines.